easybuild/easyconfigs/h/hhsuite/hhsuite-3.3.0-foss-2020b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on STAR-2.7.7a-GCC-10.2.0.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'hhsuite'
+version = '3.3.0'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/soedinglab/hh-suite/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd67f7f3bf601e48c9c0bc4cf1fbe3b946f787a808bde765e9436a48d27b0964']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Doxygen', '1.8.20'),
+]
+
+separate_build_dir = True
+
+_binaries = [
+    'hhalign', 'hhalign_mpi', 'hhalign_omp',
+    'hhblits', 'hhblits_mpi', 'hhblits_omp',
+    'hhmake',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in _binaries],
+    'dirs': ['data', 'scripts', ]
```

Comment:
@sassy-crick as a nitpick, could you clean this up, please?

Reply:
@verdurin Sorry, I am not sure what you mean, could you elaborate? Thanks!

Reply:
`'dirs': ['data', 'scripts', ] ` to `'dirs': ['data', 'scripts']`